### PR TITLE
Feature/add tflite ci testing

### PR DESCRIPTION
**Pull request by c-trejos**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -5,12 +5,13 @@ on:
     branches:
       - master
       - dev-*
+      - feature/add-tflite-ci-testing
```
Comment (michaelgruner) [inline .github/workflows/tests.yaml:8]: Remember to remove this.

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -27,3 +28,22 @@ jobs:
...
+      TENSORFLOW_PATH: /root/r2inference/backends/tflite/v2.0.1/include/tensorflow
+      CPPFLAGS: "-I${TENSORFLOW_PATH} -I${TENSORFLOW_PATH}/tensorflow/lite/tools/make/downloads/flatbuffers/include"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install backend version
+      run: |
+        cp /root/r2inference/backends/tflite/v2.0.1/binaries/libtensorflow-lite.a /usr/local/lib/    
+    - name: Configure the project
+      run: |
+        export CPPFLAGS="${{ env.CPPFLAGS }}"
```
Comment (michaelgruner) [inline .github/workflows/tests.yaml:44]: Why is this necessary?

Comment (c-trejos) [reply .github/workflows/tests.yaml]: To my understanding, when a variable is defined globally it can be referenced anywhere within the workflow, but when it comes to steps with scripts or any child process that needs such variable it needs to be explicitly passed. The export wasn't necessary, so I  removed it.

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -27,3 +27,22 @@ jobs:
...
+      TENSORFLOW_PATH: /root/r2inference/backends/tflite/v2.0.1/include/tensorflow
+      CPPFLAGS: "-I${TENSORFLOW_PATH} -I${TENSORFLOW_PATH}/tensorflow/lite/tools/make/downloads/flatbuffers/include"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install backend version
+      run: |
+        cp /root/r2inference/backends/tflite/v2.0.1/binaries/libtensorflow-lite.a /usr/local/lib/    
+    - name: Configure the project
+      run: |
+        CPPFLAGS="${{ env.CPPFLAGS }}"
```
Comment (michaelgruner) [inline .github/workflows/tests.yaml:43]: I think the variable is define job-wise, it's unnecessary here.
